src/components/common/LoginPages/sections/set-password-form.tsx

```diff
@@ -0,0 +1,117 @@
+import { useTranslation } from "react-i18next";
+import { useSyncLanguage } from "@/hooks/useSyncLanguage";
+import { FormContainer } from "@/components/common/LoginPages/layout/form-container";
+import { Heading } from "@/components/common/LoginPages/ui/heading";
+import { MyInput } from "@/components/design-system/input";
+import { MyButton } from "@/components/design-system/button";
+import { Link } from "@tanstack/react-router";
+import { setPasswordSchema } from "@/schemas/login";
+import { z } from "zod";
+import { useState } from "react";
+import { SetPasswordState } from "../types";
+import { setPassword } from "@/components/common/LoginPages/hooks/dummy/login/reset-password-button";
+
+export function SetPassword() {
+    useSyncLanguage();

```

Comment:
Add it globally

Reply:
As discussed, have removed it since we do not need localization for now.